inspo-board/src/components/BoardList.js

```diff
@@ -19,4 +20,16 @@ const BoardList = ({ boards, onSelectBoard }) => {
   );
 };
 
+BoardList.propTypes = {
+  Boards: PropTypes.arrayOf(
+    PropTypes.shape({
+      boardId: PropTypes.number.isRequired,
```

Comment:
We only need the one board_id field with the underscore in this case. At this point, the boards still have their backend object names so the name with the underscore. It is when they are passed in to the individual boards with a prop name of boardId that the name is changed. You can verify this by running the app and checking the components under inspect - board_id is present for each board in BoardList while boardId is what is displayed in the Board component. Sorry this is confusing!